src/screens/profile/addressForm.js

```diff
@@ -71,7 +93,8 @@ export default function AddressForm({
             {show ? (
                 <Modal show={show} onHide={handleCloseModal}>
                     <Modal.Header closeButton>
-                        <Modal.Title>{action === "add" ? "Add Address" : "Edit address"}</Modal.Title>
+                        <Modal.Title>{action === "add" ? (<FormattedMessage id='Add Addressess' />) :
```

Comment:
@MohdAzzam change it as 
(<FormattedMessage id={action ==  "add" ? 'Add Addressess' : "Edit address"} />